lib/src/utils/logs.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:flutter/foundation.dart';
+
+void debugPrint(String message) {
+  if (kDebugMode) {
+    print(message);
```

Comment:
Isn't [`developer.log`](https://github.com/customerio/amiapp-flutter/blob/main/lib/src/app.dart#L36) same as this? 🤔

Reply:
[developer.log](https://github.com/customerio/amiapp-flutter/blob/main/lib/src/app.dart#L36) doesn't check if the app is in debug mode but it adds alot of more context. 

So i'll add that in this method instead of print :D